example/centos/centos-9-x86_64-minimal-raw.yaml

```diff
@@ -0,0 +1,166 @@
+otk.version: "1"
+
+otk.define:
+  filesystem:
+    modifications:
+    # empty
+  packages:
+    build:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: x86_64
+        module_platform_id: c9s
+        releasever: "9"
+        repositories:
+          otk.include: "common/repositories/x86_64.yaml"
+        packages:
+          include:
+            - coreutils
+            - dosfstools
+            - glibc
+            - platform-python
+            - policycoreutils
+            - python3
+            - rpm
+            - selinux-policy-targeted
+            - systemd
+            - xfsprogs
+            - xz
+          exclude: []
+    os:
+      otk.external.osbuild-gen-depsolve-dnf4:
+        architecture: x86_64
+        module_platform_id: c9s
+        releasever: "9"
+        repositories:
+          otk.include: "common/repositories/x86_64.yaml"
+        packages:
+          include:
+            - "@core"
+            - NetworkManager-wifi
+            - dosfstools
+            - dracut-config-generic
+            - efibootmgr
+            - grub2-efi-x64
+            - initial-setup
+            - iwl3160-firmware
+            - iwl7260-firmware
+            - kernel
+            - libxkbcommon
+            - selinux-policy-targeted
+            - shim-x64
+            - xfsprogs
+          exclude: []
+  files:
+    otk.external.osbuild-gen-inline-files:
+      inline:
+        kickstart:
+          contents: |
+            # Run initial-setup on first boot
+            # Created by osbuild
+            firstboot --reconfig
+            lang en_US.UTF-8
+  kernel:
+    cmdline: ro
+    package:
+      otk.external.osbuild-get-dnf4-package-info:
+        packageset: ${packages.os}
+        packagename: "kernel"
+
+otk.include: "common/partition-table/x86_64/minimal-raw.yaml"
+
+otk.target.osbuild:
+  pipelines:
+    - otk.include: "pipeline/build/generic.yaml"
+    - name: os
+      build: name:build
+      stages:
+        - type: org.osbuild.kernel-cmdline
+          options:
+            root_fs_uuid: 9851898e-0b30-437d-8fad-51ec16c3697f
```

Comment:
This can be ` ${filesystem.const.partition_map.root.uuid}`, yes?

Reply:
Nice, force-pushed to fix.